puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging/tfm-packaging-pr-test.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- job:
+    name: tfm-packaging-pr-test
+    project-type: workflow
+    properties:
+      - github:
+          url: https://github.com/theforeman/foreman-packaging
+    triggers:
+      - github_pr_rpm_develop:
+          context: 'packaging'
```

Comment:
Will this conflict with the existing context for the current PR job?

Reply:
It will indeed. Shall we use another name for now? `obal`? `rpm`?

Reply:
I lean towards rpm for the time being